DUI3-DX/Converters/ArcGIS/Speckle.Converters.ArcGIS3/Utils/FeatureClassUtils.cs

```diff
@@ -122,7 +122,7 @@ public ACG.GeometryType GetLayerGeometryType(VectorLayer target)
       GISLayerGeometryType.POLYLINE => ACG.GeometryType.Polyline,
       GISLayerGeometryType.MULTIPATCH => ACG.GeometryType.Multipatch,
       GISLayerGeometryType.POLYGON3D => ACG.GeometryType.Multipatch,
-      _ => throw new ArgumentOutOfRangeException(nameof(target)),
+      _ => throw new ArgumentOutOfRangeException($"{originalGeomType}"),
```

Comment:
What is the first parameter of `ArgumentOutOfRangeException` is it a message, or is it an argument name?

Reply:
It looks like it is wanting the parameter name, so `nameof(target)` was correct no?
You can allways do 
```csharp
new ArgumentOutOfRangeException(nameof(target), "my custom message about what this exception is...");
```

Reply:
hmm, true, but parameter name doesn't help the user (e.g. message _"System.ArgumentOutOfRangeException: 'Specified argument was out of the range of valid values. (Parameter 'MultiPolygonZ')'"_ is more helpful than "Parameter 'target' " / "Parameter 'originalGeomType' "

Reply:
It can also be $"{nameof(target)} = {originalGeomType}"

Reply:
We have to have info for both so can we include both in the message and use nameof(<arg>) in the code, it might be a detail but if we're looking at this exception I want to know which thing in the code was out of range.  I also don't have the context, so is the argument out of range or invalid?

Reply:
@BovineOx I eventually added both!